src/containers/molecules.js

```diff
@@ -12,7 +12,8 @@ import Molecules from '../components/molecules';
 class MoleculesContainer extends Component {
 
   componentDidMount() {
-    this.props.dispatch(molecules.loadMolecules());
+    var options = { limit: 25, offset: 0, sort: '_id', sortdir: -1 }
```

Comment:
perfect const here ( options will not change ).

Reply:
Done